WooCommerce/Classes/ViewRelated/Dashboard/Analytics Hub/AnalyticsTimeRangeCard.swift

```diff
@@ -71,6 +83,27 @@ struct AnalyticsTimeRangeCard: View {
         .padding([.top, .bottom])
         .frame(maxWidth: .infinity)
     }
+
+    /// Tracks the range selection internally to determine if the custom range selection should be presented or not.
+    /// If custom range selection is not needed, the internal selection is forwarded to `selectionType`.
+    ///
+    private func internalSelectionBinding() -> Binding<AnalyticsHubTimeRangeSelection.SelectionType> {
+        .init(
+            get: {
+                selectionType
```

Comment:
I think this will be solved more elegantly once we decouple the Business options from the UI options.
I'll make sure handle that on https://github.com/woocommerce/woocommerce-ios/issues/8340

Reply:
In the mean time, I've adopted to suggestion as it does not hunt to have it right now!